github.ts

```diff
@@ -76,6 +76,32 @@ export async function queryNotifications({
   return allNotifications;
 }
 
+export async function querySearchIssues({
+  query,
+}: QueryProviderEvent): Promise<any[]> {
+  let api = await GithubApi.fromConfig();
+
+  let queryFilter = query.filter.find((f) => f.prop === "query");
+  if (!queryFilter) {
+    throw Error("No 'query' specified, this is mandatory");
+  }
+  query.filter = query.filter.filter((f) => f.prop !== "query");
```

Comment:
Using a filter to pass an argument seems uncommon for queries. Is there another way to pass an argument?

Reply:
Yeah it's a bit strange perhaps, but now the only way to do it.